docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# Output some helpful messaging when invoking `bin/start` (which itself is
```

Comment:
@jgarber623 Stole and then modified your entrypoint script. I like this better than just `tail`ing nothing

Reply:
`[ APPROVED ]`

I landed on a change like this after writing a dozen PR descriptions explaining precisely when my project’s `bin/start` script was “done” running. Finally spent the time sorting out a more obvious approach. Glad you stole it! 😄